packages/core/primitive/src/primitive.tsx

```diff
@@ -6,7 +6,7 @@ function composeEventHandlers<E>(
   return function handleEvent(event: E) {
     originalEventHandler?.(event);
 
-    if (checkForDefaultPrevented === false || !((event as unknown) as Event).defaultPrevented) {
+    if (checkForDefaultPrevented === false || !(event as unknown as Event).defaultPrevented) {
```

Comment:
Ah I guess this change will trigger the changes in versions anyway so your versions are probably correct.

Reply:
In fact I'm going to revert it just to keep things simple.